library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/service/GafferPopNamedOperationServiceTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.tinkerpop.service;
+
+import org.apache.tinkerpop.gremlin.structure.service.Service.Type;
+import org.junit.jupiter.api.Test;
+
+import uk.gov.gchq.gaffer.tinkerpop.GafferPopGraph;
+
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+
+public class GafferPopNamedOperationServiceTest {
+    private final GafferPopGraph graph = mock(GafferPopGraph.class);
+
+    @Test
+    void shouldGetType() {
+        // Given
+        GafferPopNamedOperationService<String, String> namedOpService = new GafferPopNamedOperationService<>(graph);
```

Comment:
None of the parameters have `final`. I don't think they need it, just seems to be everywhere else in Gaffer

Reply:
I don't think so either, but happy to add this.